frontend-react/package.json

```diff
@@ -91,7 +91,10 @@
     "jest": {
         "transformIgnorePatterns": [
             "[/\\\\]node_modules[/\\\\](?!(react-markdown|vfile|vfile-message|markdown-table|unist-.*|unified|bail|is-plain-obj|trough|remark-.*|rehype-.*|html-void-elements|hast-util-.*|zwitch|hast-to-hyperscript|hastscript|web-namespaces|mdast-util-.*|escape-string-regexp|micromark.*|decode-named-character-reference|character-entities|property-information|hast-util-whitespace|space-separated-tokens|comma-separated-tokens|pretty-bytes|ccount|mdast-util-gfm|gemoji)).+\\.(js|jsx|mjs|cjs|ts|tsx)$"
-        ]
+        ],
+        "moduleNameMapper": {
```

Comment:
Similar to [this solution](https://github.com/CDCgov/prime-reportstream/pull/7764#discussion_r1058621450), but we're going through the resolver rather than the transformer.  I think this'll be better since we won't have to use `axios.default` in tests anymore 🤔